configServer.lua

```diff
@@ -0,0 +1,58 @@
+svrConfig = {
+    ["webHooks"] = {
+        ['default'] = '',
```

Comment:
While we're moving these, it would be a nice touch these alphabetically.
VScode has a "sort lines" plugin which might come in handy

Reply:
Done